src/struct_ls/pfmg_setup.c

```diff
@@ -187,7 +188,7 @@ hypre_PFMGSetup( void               *pfmg_vdata,
          RT_l[l] = hypre_zPFMGCreateRestrictOp(A_l[l], cdir, stride);
       }
 #endif
-      HYPRE_StructMatrixSetTranspose(RT_l[l], 1);
+      hypre_StructMatrixSetTranspose(RT_l[l], 1, &resize);
```

Comment:
This isn't necessary here because there is no data allocated yet, i.e., there is nothing to resize.

Reply:
The reason for this change is to use the private `hypre_` API instead of the public one `HYPRE_`. Should I revert?

Reply:
Yes, I would revert.  The internal API provides the extra resize feedback (rarely needed), but you don't need it here and it confused me. :)  I realize one just calls the other but...

Reply:
Done, thanks!